input/ACTCMB.yml

```diff
@@ -0,0 +1,69 @@
+tracers:
+  ACTCMB__D56:
+    file_map:
+      - '../data/ACT_COMPSEPT/tSZ/tilec_single_tile_D56_cmb_map_v1.2.0_joint.fits'
+    file_mask:
+      - '../data/ACT_COMPSEPT/masks/act_dr4.01_s14s15_D56_compsep_mask.fits'
+    file_noise: 
+    - '../data/ACT_COMPSEPT/tSZ/tilec_single_tile_D56_cmb_map_v1.2.0_joint_noise.fits',
+    nside: 4096
+    mask_name: 'ACTCMB_D56'
+    mapper_class: 'MapperACTCMB'
+    path_lite: 'data_ACTCMB'
+    
+  ACTCMB__BN:
+  # Overlaps with BOSS
+    file_map:
+      - '../data/ACT_COMPSEPT/tSZ/tilec_single_tile_BN_cmb_map_v1.2.0_joint.fits'
+    file_mask:
+      - '../data/ACT_COMPSEPT/masks/act_dr4.01_s14s15_BN_compsep_mask.fits'
+    file_noise: 
+    - '../data/ACT_COMPSEPT/tSZ/tilec_single_tile_BN_cmb_map_v1.2.0_joint_noise.fits',
+    nside: 4096
+    mask_name: 'ACTCMB_BN'
+    mapper_class: 'MapperACTCMB'
+    path_lite: 'data_ACTCMB'
+
+cls:
```

Comment:
And nothing else. throw everything else out

Reply:
Done!